thirdparty/faiss/faiss/invlists/InvertedLists.cpp

```diff
@@ -92,6 +92,28 @@ const uint8_t* InvertedLists::get_single_code(size_t list_no, size_t offset)
     return get_codes(list_no) + offset * code_size;
 }
 
+size_t InvertedLists::get_segment_num(size_t list_no) const {
```

Comment:
let's inline these functions

Reply:
It's seems that inline virtual function is legal but gain no benefit in our case, since the function call use polymorphism mechanism determined at runtime but inline sign processed at compiler time. I find some material to support the viewpoint: https://www.geeksforgeeks.org/inline-virtual-function/